src/EditorFeatures/CSharpTest/Classification/SyntacticTaggerTests.cs

```diff
@@ -35,21 +36,34 @@ public class SyntacticTaggerTests
                     null,
                     new SyntacticClassificationTaggerProvider(null, null, null));
 
-                SnapshotSpan span = default(SnapshotSpan);
+                var expectedLength = subjectBuffer.CurrentSnapshot.Length;
```

Comment:
Ah.  Subtle.  I wanted to access `CurrentSnapshot` before the `await` in case it changed across the await boundary.